include/f_core/utils/n_gnss_utils.h

```diff
@@ -7,8 +7,8 @@
 
 namespace NGnssUtils {
      struct __attribute__((packed)) GnssCoordinates {
-        float latitude;
-        float longitude;
+        double latitude;
```

Comment:
am i misunderstanding your slack messages? i thought we said floats were more than enough - or if we're passing it around inside with max resolution and downcasting later, should altitude also be a double?

Reply:
Oh that file, should sort of be blasted now. Can't really use those types now that we use autocoders, and I don't really want a dependency between the two